x/staking/keeper/validator.go

```diff
@@ -73,16 +73,6 @@ func (k Keeper) SetValidator(ctx context.Context, validator types.Validator) err
 	return store.Set(types.GetValidatorKey(validator.GetOperator()), bz)
 }
 
-// SetValidatorByConsAddr sets a validator by conesensus address
```

Comment:
Cannot we keep this helper? Otherwise, it is way more verbose thorough the codebase.